src/browser/modules/D3Visualization/components/Graph.jsx

```diff
@@ -23,13 +23,32 @@ import {createGraph, mapNodes, mapRelationships, getGraphStats} from '../mapper'
 import {GraphEventHandler} from '../GraphEventHandler'
 import '../lib/visualization/index'
 import { dim } from 'browser-styles/constants'
+import {StyledZoomHolder, StyledSvgWrapper, StyledZoomButton} from './styled'
+import { ZoomInIcon, ZoomOutIcon } from 'browser-components/icons/Icons'
 
 export class GraphComponent extends Component {
 
+  constructor (props) {
+    super(props)
+    this.state = {}
```

Comment:
We may define state at once like this?
this.state = {
      zoomInLimitReached: true,
      zoomOutLimitReached: false
    }